### PR TITLE
MEN-4151: update the onboarding bash snippet to use get.mender.io

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Review by mzedel — APPROVED** [review]

**Comment by tranchitella** [comment]
@lluiscampos can you please double-check this one?

**Comment by lluiscampos** [comment]
> @lluiscampos can you please double-check this one?

The code looks good but I am going to give it a try on real hardware.

File: src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap [hunk 1]
```diff
@@ -74,8 +74,7 @@ exports[`DebPackage Component renders correctly 1`] = `
         }
       }
     >
-      wget https://d1b0l86ne08fsf.cloudfront.net/master/dist-packages/debian/armhf/mender-client_master-1_armhf.deb && \\
-sudo dpkg -i --force-confdef --force-confold mender-client_master-1_armhf.deb
+      wget -q -O- https://get.mender.io/ | sudo bash
```
Comment (lluiscampos) [inline src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap:77]: Here, and everywhere else, we want to use experimental for now. Something like:
```suggestion
      wget -q -O- https://get.mender.io/ | sudo bash -s -- -c experimental
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Looks good to me. I've been playing a bit in self-hosted demo and it the snippets look good.

However, as discussed in stand-up:
1) Let's make for now the GUI to use experimental. For that see my comment below. Please commit this with an independent commit, so that we can just `git revert` it later.
2) Even with 1) in place, we need to wait for merging this PR until [MEN-4037](https://tracker.mender.io/browse/MEN-4037) and a follow-up by me on `mender-dist-packages` repository.

File: src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap (hunk 1 above)
Comment (tranchitella) [reply src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap]: https://github.com/mendersoftware/gui/pull/1111/commits/19ce832a44490aeef5fb572ddcd2cbd49c36a518

**Review by lluiscampos — APPROVED** [review]

**Comment by lluiscampos** [comment]
>     2. Even with 1) in place, we need to wait for merging this PR until [MEN-4037](https://tracker.mender.io/browse/MEN-4037) and a follow-up by me on `mender-dist-packages` repository.

See https://github.com/mendersoftware/mender-dist-packages/pull/53

**Comment by lluiscampos** [comment]
`experimental` distribution is now updated with the correct dependencies. I just verified that the exact command from this PR works.

Merging.

**Comment by tranchitella** [comment]
Thanks @lluiscampos

**Pull request merged**